python/ffsim/protocols/trace_protocol.py

```diff
@@ -45,3 +51,72 @@ def trace(obj: Any, norb: int, nelec: tuple[int, int]) -> float:
         "The object did not have a _trace_ method that returned the trace "
         "or a _diag_ method that returned its diagonal entries."
     )
+
+
+def _trace_term(
+    op: tuple[tuple[bool, bool, int], ...], norb: int, nelec: tuple[int, int]
+):
+    n_elec_alpha, n_elec_beta = nelec
```

Comment:
For consistency with the rest of ffsim, use `n_alpha` and `n_beta` for these variable names.